versioned_docs/version-2.x/appendices/migration-guides/index.mdx

```diff
@@ -24,6 +24,32 @@ project, you must also update the
 composer require front-commerce/magento2-commerce-module@2.1.0
 ```
 
+### Adobe Commerce Negotiable quotes
+
+In this version, we added support for
+[Negotiable quotes](https://experienceleague.adobe.com/docs/commerce-admin/b2b/quotes/quotes.html).
+When updating your project, please note that some changes have been made on the
+following files:
```

Comment:
it's not very clear what the integrator is supposed to do with that. I guess we can advice them to keep their override in sync with the core files and if they want to use the Negotiable Quotes, they must do it. So in that regard, links to commits doing the change would be interesting.